packages/jest-expo/src/preset/expoModules.js

```diff
@@ -79,8 +79,8 @@ module.exports = {
   ExponentAR: {
     ARFaceTrackingConfiguration: { type: 'boolean', mock: false },
     ARKitVersion: { type: 'string' },
-    AROrientationTrackingConfiguration: { type: 'boolean', mock: false },
-    ARWorldTrackingConfiguration: { type: 'boolean', mock: false },
+    AROrientationTrackingConfiguration: { type: 'boolean', mock: true },
+    ARWorldTrackingConfiguration: { type: 'boolean', mock: true },
```

Comment:
I try to not modify those mocks files when I commit this file, otherwise every other person regenerating it in a different environment would always change this, status bar height, deviceYearClass etc. But that's just a suggestion!

Reply:
ah that's a good call, thanks!